pkgs/test/vim/configs.nix

```diff
@@ -0,0 +1,10 @@
+# nix-instantiate --eval --strict . -A tests.vimConfigs
+{ lib, pkgs }:
```

Comment:
pkgs is unused, right?

Reply:
this file shoundlnt have been committed, it's not used, I've just removed it. thanks